options/options.c

```diff
@@ -216,6 +217,7 @@ const struct m_sub_options vo_sub_opts = {
         .panscan = 0.0f,
         .keepaspect = 1,
         .keepaspect_window = 1,
+        .hidpi_window_scale = 1,
```

Comment:
why set default to yes? is it ok to set it default to no? I really want to not use window scale as default.